nipype/pipeline/engine/utils.py

```diff
@@ -904,7 +1142,7 @@ def _standardize_iterables(node):
     fields = set(node.inputs.copyable_trait_names())
     # Flag indicating whether the iterables are in the alternate
     # synchronize form and are not converted to a standard format.
-    synchronize = False
+    # synchronize = False  # OE: commented out since it is not used
```

Comment:
is these comments are valid for `node.synchronize` instead of `synchronize`. would be good to clean it so it's clear

Reply:
I didn't want to remove it since the comment above seemed relevant. But I agree that once we clarify the intent of this synchronize, we should clean up these comments.

Reply:
that was my guess that the these comments are relevant also for `node.synchronize` but wasn't sure if entirely.

Reply:
Added to #2320

Reply:
@effigies my concerns were more about the comments explaining `synchronize`. Does it still apply to `node.synchronize` after all changes? I don't fully understand the comments and never used it, so if you can confirm (or suggest changes) that would be great.

Reply:
Oh, sorry. It does read to me that there was intent to switch synchronize behavior with this flag, and that the two lines above only apply to that case. I think I would remove them, because the comment below seems to accurately describe the operation.